engine/src/main/java/org/terasology/world/generator/WorldGenerator.java

```diff
@@ -39,6 +42,10 @@
 
     World getWorld();
 
+    List<Zone> getZones();
+
+    Zone getNamedZone(String name);
```

Comment:
Should this be `Optional<Zone>`? Also, are we going to identify by String or by Uri?